### ISSUE TITLE
Loss does not decrease

### ISSUE THREAD

**Issue by EstherJinyan** [lzx551402/tfmatch#11]
Hi,I encountered a problem that the loss did not decrease.
Specifically,the loss remains at 0.5 from the 100th iteration to 360,000 iterations  and  the effect has not improved after training.The indicator is always worse than what you give(avg_precision=0.71,avg_MMA=0.69) with the code and data that 
 is same with you.
So,What do you think is the possible cause of this problem?

**Comment by dedoogong** [comment]
Same here. I also see that the MMA evaluation score of really less than the reported value : just 0.4~0.5!

Can @lzx551402 you please tell us the detail training sequence?

my training step is as below;
step 1: ./train_aslfeat_base.sh  --data_split blendedmvg
step 2: ./train_aslfeat_dcn.sh  --data_split blendedmvg

Do I need to use Hard Contrastive Loss in step 1?
I evaluated all of the created models per 10000 steps, but I can't see any improvement. 

Please help me!
